src/features/winterStorageApplicationView/WinterStorageApplicationView.tsx

```diff
@@ -46,9 +46,9 @@ const WinterStorageApplicationView = ({
           {t('applicationList.applicationType.newApplication')}{' '}
           {formatDate(applicationDetails.createdAt, i18n.language)}
         </Text>
-        <Chip
+        <StatusLabel
           className={styles.chip}
```

Comment:
Here's one leftover class called `chip`, and I think I saw some function or test title somewhere which had "chip" in it

Reply:
Yup, fixed